### PR TITLE
Add build info for new node version

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: alpine/aarch64/build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	echo "commit for v$NODE_VERSION not found!"
+	exit 1
+fi
+
+# compile node
+cd node \
+	&& git checkout ${commit[0]} \
+	&& ./configure --prefix=/ --shared-zlib --shared-openssl \
+   	&& make -j$(nproc) -C out mksnapshot \
+   	&& paxctl -c -m out/Release/mksnapshot \
```
Comment (petrosagg) [inline alpine/aarch64/build.sh]: Why do we need to modify PaX settings here? We're not using PaX anywhere AFAIK

Comment (nghiant2710) [inline alpine/aarch64/build.sh]: @petrosagg, https://git.alpinelinux.org/cgit/aports/tree/main/nodejs/APKBUILD?id=c6acdb11e843e65acf68d71473134149ea4604d3.
I follow the way they build for official package.

**Comment by kennethlimcp** [comment]
Will this be available soon? :)

File: alpine/aarch64/build.sh (hunk 1 above)
Comment (petrosagg) [inline alpine/aarch64/build.sh]: @nghiant2710 I don' t think we need these. This is only if you're running a Grsecurity/PaX kernel, which we don't and the project also stopped being public. https://github.com/nodejs/build/issues/75#issuecomment-188818187

**Pull request merged**